packages/create-cloudflare/src/cli.ts

```diff
@@ -51,7 +51,7 @@ const isUpdateAvailable = async () => {
 // Spawn a separate process running the most recent version of c3
 export const runLatest = async () => {
 	const args = process.argv.slice(2);
-	await runCommand(`${npm} create cloudflare@latest ${args.join(" ")}`);
+	await runCommand(`${npm} create cloudflare@latest -- ${args.join(" ")}`);
```

Comment:
We should be using package-manager-manager here to avoid worrying about this stuff: https://github.com/james-elicx/package-manager-manager/blob/main/src/commands/getRunExec.ts#L38

Reply:
Thanks Pete 😄, I thought of that too but package-manager-manager doesn't handle the create command (`getRunExec` is only used to practically run `npx`,`pnpm exec`,`pnpm dlx`, etc...) 😓, so if we were to use the library this would have to be converted to such command (I don't think that that would be too difficult or too tedious but a bit uglier than using a simple plain `create` I'd imagine)

but I'm 1000% perfect up for adding support for it there for create so that we can use the library here if we want! 🙂 😁